Gemfile

```diff
@@ -2,33 +2,39 @@ source "https://rubygems.org"
 
 gemspec
```

Comment:
Should there be a `ruby` directive in this file?

Reply:
Probably not, if we want it to work with multiple rubies.